src/Microsoft.TestPlatform.Client/DesignMode/DesignModeTestHostLauncherFactory.cs

```diff
@@ -12,12 +12,12 @@ namespace Microsoft.VisualStudio.TestPlatform.Client.DesignMode;
 /// </summary>
 public static class DesignModeTestHostLauncherFactory
 {
-    private static ITestHostLauncher s_defaultLauncher;
-    private static ITestHostLauncher s_debugLauncher;
+    private static ITestHostLauncher3 s_defaultLauncher;
```

Comment:
This is because we know what we are passing in a it is better to tell that to all those internal implementations directly, so we don't have do if is ITestHostLauncher, and if ITestHostLauncher2, when we know that it will always be ITestHostLauncher3.